spec.bs

```diff
@@ -6517,11 +6658,21 @@ An <dfn>auction config</dfn> is a [=struct=] with the following [=struct/items=]
     priorities. In the case both have entries with the same key, the entry in
     `per_buyer_priority_signals` takes precedence. No signals key start with "browserSignals.", which
     is reserved for values coming from the browser.
+  : <dfn>seller Private Aggregation coordinator</dfn>
+  :: An [=aggregation coordinator=]. Defaults to the [=default aggregation
+      coordinator=].
   : <dfn>required seller capabilities</dfn>
   :: A [=set=] of [=seller capabilities=].
     The [=seller capabilities=] that each [=interest group=] must declare to participate in the
     auction. Interest groups that don't declare all these capabilities will not participate in the
     auction.
+  : <dfn>auction report buyer keys</dfn>
+  :: A [=map=] from buyer [=origins=] to {{bigint}}s.

```

Comment:
can we add a sentence describing what each of the new fields are about?
The [explainer](https://github.com/WICG/turtledove/blob/main/FLEDGE_extended_PA_reporting.md#reporting-per-buyer-latency-and-statistics-to-the-seller) has some descriptions.